docs/src/pages/[platform]/components/menu/react.mdx

```diff
@@ -203,11 +237,13 @@ _Using style props:_
 import { Menu, MenuItem } from '@aws-amplify/ui-react';
 import '@aws-amplify/ui-react/styles.css';
 
-<Menu direction="row">
-  <MenuItem>Download</MenuItem>
-  <MenuItem>Create a Copy</MenuItem>
-  <MenuItem>Mark as Draft</MenuItem>
-</Menu>;
+export const StylePropsExample = () => (
+  <Menu direction="row">
```

Comment:
Is this enough of an example to show off the style props? Maybe style the MenuItem's as well?